Mlem/App/Views/Shared/Navigation/NavigationPage+View.swift

```diff
@@ -135,7 +135,7 @@ extension NavigationPage {
                 .disabled(requiredFeature.map { !instance.software.supports($0) } ?? false)
             } header: {
                 if let requiredFeature {
-                    Text("This feature is not available on all instances.")
+                    Text("This feature is only supported on instances running certain software versions.")
```

Comment:
Looks like update from the previous PR got missed here

```suggestion
                    Text("This feature is not available on all instances.")
```